bibliography.bib

```diff
@@ -7642,3 +7642,585 @@ @Article {PacBedCha-etal2019AL
+    editor       = {},
+    series       = {},
+    publisher    = pub_MITP,
+    address      = Cambridge_MA_USA,
+    year         = {2019},
+    edition      = {},
+    doi          = {},
+    note         = {},
+    annote       = {},
+}
+
+@Book {BoyVan2004book,
+    title        = {Convex Optimization},
+    author       = Boyd_S # and # Vandenberghe_L,
+    editor       = {},
+    series       = {},
+    publisher    = pub_CUP,
+    address      = Cambridge_GBR,
+    year         = {2004},
+    edition      = {},
+    doi          = {10.1017/CBO9780511804441},
+    note         = {},
+    annote       = {},
+}
+
+@Book {KorVyg2018book,
+    title        = {Combinatorial Optimization: Theory and Algorithms},
+    author       = Korte_B # and # Vygen_J,
+    editor       = {},
+    series       = srs_AC,
+    publisher    = pub_SP,
+    address      = Berlin_DEU,
+    year         = {2018},
+    edition      = {Sixth},
+    doi          = {10.1007/978-3-662-56039-6},
+    note         = {},
+    annote       = {},
+}
+
+@Article {EdmKar1972JACM,
+    title        = {Theoretical improvements in algorithmic efficiency for network flow problems},
+    author       = Edmonds_J # and # Karp_RM,
+    journal      = JACM,
+    series       = {},
+    volume       = {19},
+    number       = {2},
+    year         = {1972},
+    pages        = {248--264},
+    doi          = {10.1145/321694.321699},
+    note         = {},
+    annote       = {},
+}
+
+@Article {LinKer1973OPRE,
+    title        = {An Effective Heuristic Algorithm for the Traveling-Salesman Problem},
+    author       = Lin_S # and # Kernighan_BW,
+    journal      = OPRE,
+    series       = {},
+    volume       = {21},
+    number       = {2},
+    year         = {1973},
+    pages        = {498--516},
+    doi          = {10.1287/opre.21.2.498},
+    note         = {},
+    annote       = {},
+}
+
+@Article {PadRin1991SIAMR,
+    title        = {A branch-and-cut algorithm for the resolution of large-scale symmetric traveling salesman problems},
+    author       = Padberg_M # and # Rinaldi_G,
+    journal      = SIAMR,
+    series       = {},
+    volume       = {33},
+    number       = {1},
+    year         = {1991},
+    pages        = {60--100},
+    doi          = {10.1137/1033004},
+    note         = {},
+    annote       = {},
+}
+
+@InCollection {Dan1990history,
+    title        = {Origins of the simplex method},
+    author       = Dantzig_GB,
+    booktitle    = {A history of scientific computing},
+    editor       = Nash_SG,
+    series       = {},
+    volume       = {},
+    number       = {},
+    year         = {1990},
+    pages        = {141--151},
+    publisher    = pub_ACM,
+    address      = NewYork_NY_USA,
+    doi          = {10.1145/87252.88081},
+    note         = {},
+    annote       = {},
+}
+
+@TechReport {LegGarKuc-etal2022techrep,
+    title        = {Tycho: a robust, {ROS}-based tracking system for robot swarms},
```

Comment:
```suggestion
    title        = {{Tycho}: a robust, {ROS}-based tracking system for robot swarms},
```

Reply:
In this PR, other proper names are not protected (e.g., Crazyfly, Crazyswarm, etc). Please revise.